CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ This file documents all notable changes to the GEOS-Chem Classic wrapper reposit
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased] - TBD
+### Changed
+- CMake now throws an error if FAST-JX is used with any other mechanism than Hg
```

Comment:
There should be a GEOS-Chem PR that goes with this to remove the option for FAST-JX except for mercury.

Reply:
Thanks @lizziel.  I'm not sure how to implement a GEOS-Chem PR that would prevent use of FAST-JX in the GEOS-Chem repo.  There is no good way to compare strings in preprocessor statements (i.e. `#if MECH == "Hg"` won't work).

What I could do is to remove the FAST-JX CMake compile option and then set `FASTJX=1` if `-DMECH=Hg` is passed at the command line.  This would disable FAST_JX for all mechanisms except Hg.

Reply:
I would like to keep the compile option so that it is easy for Hg developers to switch between the two. What I was thinking was put an error message within a FASTJX ifdef that if not a mercury simulation then fail. We need this for uses of GEOS-Chem outside of the super-projects.

Reply:
To further clarify, this would use the Input_Opt logical for Hg simulation, not the compile-time MECH setting.